src/main.rs

```diff
@@ -373,6 +395,15 @@ fn eprint_nothing_searched() {
                Try running again with --debug.");
 }
 
+fn print_stats(match_count: u64, paths_searched: u64, paths_matched: u64, time_elapsed: Duration) {
```

Comment:
All lines of source code in ripgrep should be wrapped to 79 columns inclusive (excepting URLs or other things that are inconvenient to wrap). Please see examples of other function declarations for formatting tips.